content/docs/install-pack.md

```diff
@@ -35,10 +35,14 @@ brew install pack
 ### Linux
 
 ```bash
-wget https://github.com/buildpacks/pack/releases/download/v{{< latest >}}/pack-v{{< latest >}}-linux.tgz
-tar xvf pack-v{{< latest >}}-linux.tgz
-rm pack-v{{< latest >}}-linux.tgz
-./pack --help
+{
```

Comment:
This command is pretty gnarly and requires that you have `wget` and `jq` preinstalled. I'd rather we do what is suggested for windows and just point them to the latest release page.